src/app/accounts/add-earn.ts

```diff
@@ -18,6 +17,8 @@ import {
 } from "@services/mongoose"
 import { AccountsIpRepository } from "@services/mongoose/accounts-ips"
 
+import { intraledgerPaymentSendWalletIdForBtcWallet } from "../payments/send-intraledger"
```

Comment:
This does not remove the circular dependency. The issue is payment -> accounts -> payments. 
By changing the import like this you are:
- importing something that we are treating as module-private
- circumventing our tracing setup